backend/eventloop/application_event_loop/application_event_runner_test.go

```diff
@@ -778,13 +779,17 @@ var _ = Describe("ApplicationEventLoop Test", func() {
 			err = gzipWriter.Close()
 			Expect(err).To(BeNil())
 
+			reconciledStateString, _, err := dummyApplicationComparedToField()
+			Expect(err).To(BeNil())
+
 			applicationState := &db.ApplicationState{
 				Applicationstate_application_id: deplToAppMapping.Application_id,
 				Health:                          string(managedgitopsv1alpha1.HeathStatusCodeHealthy),
 				Sync_Status:                     string(managedgitopsv1alpha1.SyncStatusCodeSynced),
 				Revision:                        "abcdefg",
 				Message:                         "Success",
 				Resources:                       buffer.Bytes(),
+				ReconciledState:                 reconciledStateString,
```

Comment:
Also in this test, we should check that the GitOpsDeployment's status field has the correct ReconciledState values, similar to how we are checking other status values.